app/main.py

```diff
@@ -1,3 +1,22 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        if days % 7 == 0:
+            return days / 7
+        return (days // 7) + 1
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        for key in course_dict:
+            if key == "name":
+                cls.name = course_dict["name"]
```

Comment:
Why do you create class attribute 'name'?! You don't need to store this values, you can passes this values right to 'cls()'